forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -1037,6 +1036,16 @@ export const RecordsLog = ({
     }));
   }
 
+  //function to manage download for harms option
+  const enableHarmsDonwnload = () => {
+    return !recordsObj.records.every(record => record.isredactionready || (!record.isredactionready && (record.failed || isrecordtimeout(record.created_at, RECORD_PROCESSING_HRS))));
```

Comment:
can simplify to record.isredactionready || record.failed || isrecordtimeout(record.created_at, RECORD_PROCESSING_HRS). 
Or if you want to make logic clear can use if/else

Reply:
Adjusted. Thanks for the feedback!